examples/amazonlinux2023.yaml

```diff
@@ -0,0 +1,14 @@
+# This template requires Lima v0.17.0 or later and requires macOS >=13.0 
+vmType: "vz"
+mountType: "virtiofs"
+images:
+- location: "https://cdn.amazonlinux.com/al2023/os-images/2023.5.20240805.0/kvm/al2023-kvm-2023.5.20240805.0-kernel-6.1-x86_64.xfs.gpt.qcow2"
```

Comment:
Are these images persistent for years, or are they wiped periodically?
In the latter case, the "latest" image has to be specified as a fallback option.
(See other templates)

Reply:
> Are these images persistent for years, or are they wiped periodically?

I don't know, that's an Amazon question, sorry. To my knowledge there is no `latest` option.

Reply:
> Are these images persistent for years, or are they wiped periodically?

I think "somebody" will have to reach out to AWS to find out. I'll be offline soon, so I can't do it right now; ping me in September if it is still unresolved by then. 😄 

Maybe ask the `finch` people to find out who can help. They have a channel on the CNCF Slack.

Reply:
It _could_ be left to just go 404 when it is removed, like with the other distributions?

The "latest" is a nice to have / recovery, but ultimately they all need bumping anyway.